hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatmap.tsx

```diff
@@ -66,60 +117,61 @@ export class Heatmap extends React.Component<Record<string, object>, ITreeState>
       entityType: 'key',
       inputPath: '/',
       date: '24H',
```

Comment:
I am seeing some of the values are used frequently through out the code like '24H', 'key' , '/' etc. Is it possible to define them as global constants and use them ?